lib/DocumentRetriever.ts

```diff
@@ -179,7 +179,41 @@ DocumentRetriever.prototype.getRequest = async function(this: DocumentRetriever)
 		object.ExclusiveStartKey = Document.isDynamoObject(this.settings.startAt) ? this.settings.startAt : this.internalSettings.model.Document.objectToDynamo(this.settings.startAt);
 	}
 	if (this.settings.attributes) {
-		object.ProjectionExpression = this.settings.attributes.join(", ");
+		const expressionAttributeNames = {};
+		const projectionExpression = [];
+		// Check object ExpressionAttributeNames
```

Comment:
Not sure this comment adds any value. I'm all for adding code comments, but can we try to add a bit more detail here to make it more meaningful and not just add unnecessary comments to the code base.